components/ParameterForm.vue

```diff
@@ -118,27 +126,36 @@ const renderAsRadios = (parameter: Parameter) => {
   return parameter.parameterType === ParameterType.Select && optionsAreTerse(parameter);
 };
 
-const submitForm = () => {
-  // Not implemented yet
-};
+const formSubmitting = ref(false);
 
-const largeScreen = ref(true);
-const breakpoint = 992; // CoreUI's "lg" breakpoint
-const setFieldSizes = () => {
-  if (window.innerWidth < breakpoint) {
-    largeScreen.value = false;
-  } else {
-    largeScreen.value = true;
-  }
+const submitForm = async () => {
+  if (!formData.value) {
+    return;
+  };
+
+  formSubmitting.value = true;
+  const response = await $fetch<NewScenarioData>("/api/scenarios", {
+    method: "POST",
+    query: { // Using query instead of body because I couldn't work out how to send a body in the integration test.
```

Comment:
should defs use body here... your code shouldn't change to accommodate the tests...

Reply:
I don't have a reason to prefer body over query; do you?

Reply:
Semantically the use of query params is incorrect here... they are more for filtering, sorting / picking certain ids... when doing post request and sending data its semantically correct to use a body

Reply:
I tried for some time to use the nuxt test utils to pass through a body, but there didn't seem to be a way.
To be clear, the R API is still receiving a body, it's only the web app that's taking a query parameter.

Reply:
I tried again today with a fresh brain and am making headway :)